index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  
+  <body>
+    <nav id="navbar" class="nav">
+      <ul class="nav-list">
+        <li><a href="#welcome-section">About</a></li>
+        <li><a href="#projects">Work</a></li>
+        <li><a href="#contact">Contact</a></li>
+        
+      </ul>
+    </nav>
+        
+    <section id="welcome-section">
+    <h1> Hi we are Israel and Matias</h1>
+<p>we are Front-End developers</p>
+<section id="projects">
+<h2 class="project-title"> Projects </h2>
+<a href="https://codepen.io/freeCodeCamp/full/qRZeGZ" target="_blank" class="project project-tile" id="profile-link">
+          <img class="project-image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/random-quote-machine.png" alt="project">
+          <p class="project-title">
+            <span class="code">&lt;</span>
+            Random Quote Machine
+            <span class="code">/&gt;</span>
+          </p>
+        </a>
+</section> 
+</section>
+
+
+    
```

Comment:
Also if you could please always delete unused blank space and unused comments in the html section will be nice for having a clear code !
the rest is perfect ! Congratulation 🎊